docs/differences.md

```diff
@@ -17,3 +17,4 @@ it still has some edge case behaviors. They are listed below:
 | Block hash computation | keccak256(RLP.encode(Block Header))                          | [pedersen(Starknet Block Header)](https://docs.starknet.io/documentation/architecture_and_concepts/Network_Architecture/header/)                                                                               |
 | State root computation | Root of state keccak-MPT                                     | [poseidon(root of contract pedersen-MPT, root of contract class pedersen-MPT)](https://docs.starknet.io/documentation/architecture_and_concepts/Network_Architecture/starknet-state/)                          |
 | Transaction receipt    | Transaction receipt is available once a transaction is mined | Transaction receipt is available once a transaction is processed and guaranteed to be mined. This means that blockhash field of the receipt is optional and will be non-null only once the transaction is mined. |
+| RIP-7212 Support   | Kakarot Supports native P256 verification. | P256 is a wildly used curve and it is and supported in many modern devices such as Apple’s Secure Enclave, Webauthn, Android Keychain which can be used in many use cases. |
```

Comment:
The current format of this section is to report the state of L1 Ethereum and then the state of Kakarot,

I think we should put smth like P256 verification is not supported on L1 and then link to some site that can tell ppl when it'll be included in a hardfork

and then tell about kakarot supporting it

Reply:
It makes sense, but there's no timeline or commitment about when EIP-7212 will be included. So, I'd suggest that we can add something like; 
- RIP-7212 is the first rollup improvement proposal that has been adopted by the biggest rollups
- And Kakarot is one of them. 
- There's a chance for L1 to add support to RIP-7212 as well but it is not fully decided by Ethereum governance. 

How does it sounds?

Reply:
Yes sounds good ser sorry for no reply